nbs/API/frontmatter.ipynb

```diff
@@ -17,8 +17,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def insertfm(nb, fm): nb.cells.insert(0, mk_cell(dict2fm(fm), 'raw'))
</pre><p>I use this in <code>migrate.ipynb</code> so brought this outside the processsor </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>